gradle/libs.versions.toml

```diff
@@ -0,0 +1,40 @@
+metadata.format.version = "1.1"
+
+[versions]
+
+indra = "2.2.2"
+lombok = "1.18.30"
+adventure = "4.15.0-20231207.074016-23"
```

Comment:
use version refs for everything please, as per geyser

Reply:
Why btw?

Reply:
It keeps the versions in a single place. It's the convention of every version catalogue in this organization.

Reply:
Alright. Weird convention though. I'd rather keep the version close to the artifact I retrieve. Will do this in the morning.